weaviate/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   # of weaviate. In accordance with Infra-as-code, you should pin this value
   # down and only change it if you explicitly want to upgrade the Weaviate
   # version.
-  tag: 0.22.7
+  tag: 0.22.20-6f9e35e
```

Comment:
ok for now, but we shouldn't forget to update that to `0.23.0` once we make a release here.